openapi/downstairs-repair.json

```diff
@@ -5,39 +5,11 @@
     "version": "1"
   },
   "paths": {
-    "/extent/{eid}/data": {
-      "get": {
-        "summary": "Stream the contents of the data file for the given extent ID",
-        "operationId": "get_extent",
-        "parameters": [
-          {
-            "in": "path",
-            "name": "eid",
-            "required": true,
-            "schema": {
-              "type": "integer",
-              "format": "uint32",
-              "minimum": 0
-            },
-            "style": "simple"
-          }
-        ],
-        "responses": {
-          "default": {
-            "description": "",
-            "content": {
-              "*/*": {
-                "schema": {}
-              }
-            }
-          }
-        }
-      }
-    },
-    "/extent/{eid}/db": {
+    "/extent/{eid}/files": {
       "get": {
-        "summary": "Stream the contents of the metadata .db file for the given extent ID",
-        "operationId": "get_db",
+        "summary": "For a given extent, return a vec of strings representing the names of",
+        "description": "the files that exist for that extent.",
```

Comment:
Just note that the first line  is presumed to be the terse summary. It's not a big deal for this kind of internal API, but you might want to change the comment to something like:

```rust
/**
 * Get the list of files related to an extent.
 *
 * For a given extent, return a set of strings representing the names of
 * the files that exist for that extent.
 */
```